src/algorithms/groundstate/gradient_grassmann.jl

```diff
@@ -22,13 +22,14 @@ with a preconditioner to induce the metric from the Hilbert space inner product.
 - `tol::Float64`: tolerance for convergence criterium
 - `maxiter::Int`: maximum amount of iterations
 - `verbosity::Int`: level of information display
+- `alg::OrthogonalFactorizationAlgorithm` : algorithm to perform the orthogonal factorizations during the automatic differentiation
```

Comment:
automatic differentiation?

Reply:
I ran the formatter and changed this line to : 
`alg::OrthogonalFactorizationAlgorithm` : specifies which algorithm is used for singular
  value decompositions in the retract and transport functions. Choices are SDD() which is faster but
  less stable and SVD() which is slower but more stable.

Reply:
I think if the type specifyer is there, it's probably not a great idea to also list the options, as this can change in the future (think `SafeSVD`), in which case we have to simply remember to update the docstring, which is unlikely to happen.

Reply:
That's a good point. I'll resolve this tomorrow.